spec/models/exception_hunter/error_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+require 'byebug'
+module ExceptionHunter
+  RSpec.describe ExceptionHunter::Error, type: :model do
```

Comment:
Does the `ocurred_at` get set when the error is created? Good chance to TDD and find out